Formula/aom.rb

```diff
@@ -0,0 +1,27 @@
+class Aom < Formula
+  desc "Alliance for Open Media Codec SDK"
+  homepage "http://aomedia.org"
+  head "https://aomedia.googlesource.com/aom.git"
```

Comment:
We can't accept this. I see that there is a `v1.0.0` tag which, if it is considered “stable” by upstream, we can use as a tagged version.